src/MySqlConnector/Core/DiagnosticListenerExtensions.cs

```diff
@@ -0,0 +1,323 @@
+using System;
+using System.Data;
+using System.Diagnostics;
+using System.Runtime.CompilerServices;
+using MySql.Data.MySqlClient;
+
+namespace MySqlConnector.Core
+{
+	/// <summary>
+	/// Extension methods on the DiagnosticListener class to log MySqlCommand data
+	/// </summary>
+	internal static class DiagnosticListenerExtensions
+	{
+		public const string DiagnosticListenerName = "MySqlClientDiagnosticListener";
+
+		private const string MySqlClientPrefix = "System.Data.MySqlClient.";
+
+		public const string SqlBeforeExecuteCommand = MySqlClientPrefix + nameof(WriteCommandBefore);
+		public const string SqlAfterExecuteCommand = MySqlClientPrefix + nameof(WriteCommandAfter);
```

Comment:
The guide recommends "Start" and "Stop", not "Before" and "After".

> DO - use the 'Start' and 'Stop' suffixes for events that define an interval of time.

Reply:
Fixed